### DISCUSSION TITLE
Not allowed to destroy in the wild

### DISCUSSION THREAD

**Discussion by h4ppyboy** [TownyAdvanced/Towny#7007] [General]
How do i fix this, i made all necessary changes in config
![image](https://github.com/TownyAdvanced/Towny/assets/75522441/78205e52-a9ca-4058-b368-4c91f52c21a1)

**Comment by LlmDl** [comment]
https://github.com/TownyAdvanced/Towny/wiki/Frequently-Asked-Questions
